### PR TITLE
Created article View based on person

### PR DESCRIPTION
**Pull request by prabu-ela**

### CODE REVIEW

File: config/views.view.articles_based_on_person.yml [hunk 1]
```diff
@@ -0,0 +1,242 @@
...
+status: true
+dependencies:
+  config:
+    - core.entity_view_mode.node.teaser
+    - node.type.article
+  module:
+    - node
+    - user
+id: articles_based_on_person
+label: 'Articles Based on Person'
```
Comment (malabya) [inline config/views.view.articles_based_on_person.yml]: `Author Articles`

File: config/views.view.articles_based_on_person.yml [hunk 2]
```diff
@@ -0,0 +1,242 @@
...
+    cache_metadata:
+      max-age: -1
+      contexts:
+        - 'languages:language_content'
+        - 'languages:language_interface'
+        - url.query_args
+        - 'user.node_grants:view'
+        - user.permissions
+      tags: {  }
+  page_1:
```
Comment (malabya) [inline config/views.view.articles_based_on_person.yml]: We don't need a page for this. This will be a block.

**Comment by malabya** [comment]
@prabu-ela The view should be a block and placed in the Person entity view config via layout builder.

File: config/views.view.articles_based_on_person.yml (hunk 1 above)
Comment (prabu-ela) [reply config/views.view.articles_based_on_person.yml]: View name renamed to Author Articles

File: config/views.view.articles_based_on_person.yml (hunk 2 above)
Comment (prabu-ela) [reply config/views.view.articles_based_on_person.yml]: The View page is converted to view block

**Pull request merged**